tests/testsuite/message_format.rs

```diff
@@ -108,7 +113,13 @@ fn cargo_renders_ansi() {
 
     p.cargo("check --message-format json-diagnostic-rendered-ansi")
         .with_status(101)
-        .with_stdout_contains("[..]\\u001b[38;5;9merror[..]")
+        .with_stdout_data(str![[r#"
```

Comment:
ditto, for JSON messages we keep it as-is.

Reply:
addressed at commit a1f4d03